sphinx_gallery/scrapers.py

```diff
@@ -492,7 +498,7 @@ def _single_line_sanitize(s):
    :class: sphx-glr-single-img
 """
 
-# keep around for back-compat:
+# Used for SVGs
 SINGLE_IMAGE = """
  .. image:: /%s
```

Comment:
... so this is the bug, not with your code but with code we no longer use anymore
```suggestion
.. image:: /%s
```

Reply:
Ahh, thanks! Sorry for taking this long to come back to it. I sort of assumed that line was correct...